packages/renderer-commonmark/src/index.ts

```diff
@@ -53,33 +53,67 @@ function getNumberOfRequiredBackticks(text: string) {
   }, 1);
 }
 
-export default class CommonmarkRenderer extends Renderer {
+function render(renderer: CommonMarkRenderer, node: HIRNode, parent?: HIRNode, index?: number): string {
+  if (index > 0) {
+    node.previous = parent.children[index - 1];
+  }
+  if (parent && index < parent.children.length) {
+    node.next = parent.children[index + 1];
+  }
```

Comment:
v.v.minor - since not sure if you want to avoid `undefined` (and be more explicit with `null`) when out of range?